supernode/server/router.go

```diff
@@ -22,11 +22,12 @@ import (
 	"net/http"
 	"net/http/pprof"
 
-	"github.com/dragonflyoss/Dragonfly/apis/types"
-	"github.com/dragonflyoss/Dragonfly/version"
 	"github.com/gorilla/mux"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+
+	"github.com/dragonflyoss/Dragonfly/apis/types"
```

Comment:
The self-project import path may be better in second  part.

Reply:
done